public/app/features/dashboard/components/DashboardSettings/GeneralSettings.tsx

```diff
@@ -167,6 +167,11 @@ export function GeneralSettingsUnconnected({
     setRenderCounter(renderCounter + 1);
   };
 
+  const onHideIirisBreadcrumbChange = (event: any) => {
+    dashboard.hideIirisBreadcrumb = event.target.checked;
+    setRenderCounter(renderCounter + 1);
```

Comment:
What's this for?

Reply:
It is the handler for the checkbox in Grafana settings-view.

Reply:
I meant the renderCounter specifically. Seems like it reads some variable and increments it via setter function which seems a bit strange. 😄

Reply:
Ok, I don't know, it was in other handlers so I added it here also. I suppose it is used to re-render the dashboard when the settings are changed.

Reply:
I thought it might be something like that but still implementation seems weird - I would've made function like shouldReRender() which can internally do something like this. 😄